version.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
+  "version": "17.3",
+  "assemblyVersion": "17.3",
```

Comment:
TODO: This needs to be updated along with Versions.props. Determine if there is a simple way to do that. Otherwise, comment at these locations and update the New Branch Checklist items.

Reply:
Updated branch-checklist.md and added comment to Versions.props.